src/blueapi/service/interface.py

```diff
@@ -76,6 +79,29 @@ def stomp_client() -> StompClient | None:
         return None
 
 
+@cache
+def numtracker_client() -> NumtrackerClient | None:
+    conf = config()
+    if conf.numtracker is not None:
```

Comment:
Could: If there is config for numtracker but no config for metadata, it can never work, could add an if statement here to check and give a nice error message in this case.

Reply:
Done with test.